### ISSUE TITLE
Crash from null player in bonemeal handler

### ISSUE THREAD

**Issue by ProgrammerHero** [MrComputerGhost/Runic-Dungeons#11]
The Runic Dungeon (1.1.2b) noTrees event handler is assuming every bonemeal event has an attached player.  I encountered a server crash while roaming around in the twilight forest when one of the twilight forest blocks posted a bonemeal event with a null player object.  The TF block is posting the event directly (not me), so there is no player it can attach to the event.

```
Caused by: java.lang.NullPointerException
    at mrcomputerghost.runicdungeons.events.RunicEventHandler.noTrees(RunicEventHandler.java:71) ~[RunicEventHandler.class:?]
    at cpw.mods.fml.common.eventhandler.ASMEventHandler_527_RunicEventHandler_noTrees_BonemealEvent.invoke(.dynamic) ~[?:?]
    at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54) ~[ASMEventHandler.class:?]
    at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:138) ~[EventBus.class:?]
    at net.minecraft.item.ItemDye.applyBonemeal(ItemDye.java:143) ~[acj.class:?]
    at twilightforest.block.BlockTFUberousSoil.func_149695_a(BlockTFUberousSoil.java:119) ~[BlockTFUberousSoil.class:?]
    at net.minecraft.world.World.func_147460_e(World.java:697) ~[ahb.class:?]
    at net.minecraft.world.World.func_147459_d(World.java:651) ~[ahb.class:?]
    at net.minecraft.world.World.func_147444_c(World.java:613) ~[ahb.class:?]
    at net.minecraft.world.World.markAndNotifyBlock(World.java:488) ~[ahb.class:?]
    at net.minecraft.world.World.func_147465_d(World.java:466) ~[ahb.class:?]
    at net.minecraft.world.World.func_147449_b(World.java:600) ~[ahb.class:?]
...
```

**Comment by MrComputerGhost** [comment]
Will be fixed in 1.1.3, should be out tomorrow. Thank you.

**Comment by MrComputerGhost** [comment]
Fixed as of 1.1.3
